src/GitHub.InlineReviews/Tags/ShowInlineCommentTag.cs

```diff
@@ -14,20 +15,21 @@ public class ShowInlineCommentTag : InlineCommentTag
         /// Initializes a new instance of the <see cref="ShowInlineCommentTag"/> class.
         /// </summary>
         /// <param name="session">The pull request session.</param>
-        /// <param name="thread">A model holding the details of the thread.</param>
-        public ShowInlineCommentTag(
-            IPullRequestSession session,
-            IInlineCommentThreadModel thread)
-            : base(session, thread.LineNumber, thread.DiffLineType)
+        /// <param name="lineNumber"></param>
```

Comment:
Could we get a comment here? Especially for `lineNumber` parameters it's very useful to know if they're 0 or 1-based.

Reply:
Would you happen to know if it is 0 or 1 based? lol

Reply:
:+1: